Eatery/Controllers/BRB/BRBLoginViewController.swift

```diff
@@ -95,6 +98,13 @@ class BRBLoginViewController: UIViewController {
         passwordTextField.text = nil
         netidTextField.resignFirstResponder()
         passwordTextField.resignFirstResponder()
+        DefaultsKeys.updateFoodLocations {
+            DispatchQueue.main.async {
+                self.tableView.reloadData()
+            }
+        }
+        favoriteItems = Defaults[\.favoriteFoods]
+        tableView.reloadData()
```

Comment:
It seems like you expect line 103 to execute before 106-7, but that might not happen. Would that be buggy?